scripts/sql/61_alter_ci_template.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ci_template
```

Comment:
check file number once you merge into main.

Reply:
Sure